README.md

```diff
@@ -22,22 +22,22 @@ repositories {
   mavenCentral()
 }
 dependencies {
-  testImplementation 'app.cash.turbine:turbine:1.0.0'
+  testImplementation("app.cash.turbine:turbine:1.0.0")
```

Comment:
You forget to change the language from `groovy` to `kotlin` in line 20 🤓

Reply:
Changed!